echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/eventhandlers/GitEventHandler.java

```diff
@@ -43,27 +42,25 @@
  */
 @Component
 @Slf4j
-public class GitEventMonitor extends TriggerMonitor {
-
-  public static final String GIT_TRIGGER_TYPE = "git";
+public class GitEventHandler extends BaseTriggerEventHandler<GitEvent> {
+  private static final String GIT_TRIGGER_TYPE = "git";
 
   private static final String GITHUB_SECURE_SIGNATURE_HEADER = "X-Hub-Signature";
 
+  private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Echo has a ton of object mappers everywhere. In other services we _try_ to initialize one object mapper as a bean in the config so that everyone uses the same object mapper. This tends to make it easier to debug serialization issues down the road, although we're not great at actually only having one object mapper. 

As you refactor echo, it might be good to only have one object mapper instead of initializing them everywhere. For this PR, could you just initialize the object mapper in `BaseTriggerEventHandler`?

Reply:
Good point; it was bothering me that I needed all of these.  I've done two things:
* I moved the actual object conversion to the base class (which I'd wanted to do all along but was running into type-safety issues when I was trying last week, but found a nice way of doing today)
* I added the `objectMapper` as a constructor parameter of the handlers and autowired it up so we're not creating them ad hoc anymore.